wrapping/transformix_extras.py

```diff
@@ -32,3 +32,28 @@ def transformix_jacobian(*args, **kwargs):
 # Satify itk package lazy loading
 def transformix_jacobian_init_docstring():
     pass
+
+def transformix_pointset(*args, **kwargs):
+    reduce_output = kwargs.get('reduce_output', False)
+    if reduce_output:
+        kwargs.pop('reduce_output')
+    transformix_object = itk.TransformixFilter.New(*args, **kwargs)
+    transformix_object.UpdateLargestPossibleRegion()
+    dir = kwargs.get('output_directory', None)
```

Comment:
We want to use a different variable name here because `dir` is a Python built-in function.